scala/sag_desc.par

```diff
@@ -34,15 +34,15 @@ Zeus' lightning bolt crossed
 the key to the underworld
 |(
 u
-5:7-kleisma (~11:13k) (5120/5103)
+5/7-kleisma (~11/13k) (5120/5103)

```

Comment:
The way we use directed comma names here, depends on the outcome of the ongoing discussion here:
http://forum.sagittal.org/viewtopic.php?f=12&t=492

Reply:
Yes, understood.